client/components/open/components/RecordOperations.vue

```diff
@@ -44,18 +44,18 @@ export default {
   },
   methods: {
     onDeleteClick () {
```

Comment:
When they are multiple use of `useAlert`, we can just return alert from `setup` function once and use it to avoid initiating it multiple times